tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -233,45 +299,90 @@
    },
    "outputs": [],
    "source": [
-    "if credential_manager:\n",
-    "    with OLIApi(credential_manager) as oliapi:\n",
-    "        # create a new DBS file\n",
-    "        dbs_file_id = oliapi.get_dbs_file_id(\n",
-    "            chemistry_source=source_water[\"components\"],\n",
-    "            phases=[\"liquid1\", \"solid\"],\n",
-    "        )\n",
-    "        \n",
-    "        # create water analysis inputs\n",
-    "        water_analysis_input = f.build_flash_calculation_input(\n",
-    "            \"wateranalysis\",\n",
-    "            state_vars,\n",
-    "        )\n",
-    "    \n",
-    "        # create water analysis case\n",
-    "        water_analysis_base_case = f.run_flash(\n",
-    "            \"wateranalysis\",\n",
-    "            oliapi,\n",
-    "            dbs_file_id,\n",
-    "            water_analysis_input,\n",
-    "            file_name=\"water_analysis_singlepoint\"\n",
-    "        )\n",
-    "        \n",
-    "        # create apparent composition from water analysis output\n",
-    "        water_analysis_apparent_composition = f.build_flash_calculation_input(\n",
-    "            \"isothermal\",\n",
-    "            state_vars,\n",
-    "            water_analysis_base_case[0],\n",
-    "        )\n",
-    "        \n",
-    "        # run isothermal flash for the survey parameters\n",
-    "        isothermal_survey_result = f.run_flash(\n",
-    "            \"isothermal\",\n",
-    "            oliapi,\n",
-    "            dbs_file_id,\n",
-    "            water_analysis_apparent_composition,\n",
-    "            survey,\n",
-    "            \"isothermal_composition_survey\",\n",
-    "        )"
+    "# Flash calculation example #1: Water Analysis survey with pH reconciliation.\n",
+    "\n",
+    "# OLIApi is used as a context manager\n",
+    "with OLIApi(credential_manager, interactive_mode=interactive_mode) as oliapi:\n",
+    "    # create a new DBS file\n",
+    "    # alternative thermo_frameworks and databanks are available.\n",
+    "\n",
+    "    # use keep_file = True to save a DBS file ID on the Cloud\n",
+    "    # for use in more than one session\n",
+    "    dbs_file_id = oliapi.generate_dbs_file(\n",
+    "        inflows=inflows,\n",
+    "        thermo_framework=\"MSE (H3O+ ion)\",\n",
+    "        phases=[\"liquid1\", \"solid\"],\n",
+    "        model_name=\"test\",\n",
+    "        keep_file=True,\n",
+    "    )\n",
+    "    # NOTE: liquid2 (non-aqueous) and vapor phases are available but not yet fully supported.\n",
+    "\n",
+    "    # if a DBS file has been retained from a previous session,\n",
+    "    # its flash history and chemistry information can be summarized.\n",
+    "    file_summary = oliapi.get_dbs_file_summary(dbs_file_id)\n",
+    "\n",
+    "    # save chemistry information\n",
+    "    chemistry_info = file_summary[\"chemistry_info\"]\n",
+    "    write_output(chemistry_info[\"result\"], \"chemistry_info\")\n",
```

Comment:
This bit on getting the summary/flash history should be shown later, after walking through the main essentials.